app/views/shared/_logout.html.erb

```diff
@@ -0,0 +1,5 @@
+<% @current_user = User.where(id: session[:user_id]).first %>
```

Comment:
Do we need to find the `current_user` here? OR Have we already access the `@current_user`